proto/api/v1/remote_runner.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+
+package api.v1;
+
+import "google/rpc/status.proto";
+import "proto/remote_execution.proto";
+
+message RemoteRunRequest {
+  // URL of the repo the workflow is running for
+  // Ex. "https://github.com/some-user/acme"
+  string repo = 1;
+
+  // Git refs at which the workflow should be run (at least one of `branch` or
+  // `commit_sha` must be set). If only `branch` is set, will run from the tip
+  // of the branch. If only `commit_sha` is set, reporting will not contain the
+  // branch name.
+  string branch = 2;
+  string commit_sha = 3;
+
+  // Bash commands to run, in order.
+  // Ex. [{"run": "bazel build :server"}, {"run": "echo \"Hello World\""}]
+  repeated Step steps = 4;
+
+  // Environment variables to set in the runner env.
+  map<string, string> env = 5;
+
+  // Platform properties to apply to the runner.
+  // Ex. [{"name":"OSFamily","value":"linux"}, {"name":"Arch","value":"amd64"}]
+  repeated build.bazel.remote.execution.v2.Platform.Property
```

Comment:
I wonder if we should just do map<string, string> to make this easier to consume.

Reply:
Ack - changed